docker-compose.golang.yml

```diff
@@ -7,8 +7,15 @@ services:
       context: .
       dockerfile: golang/Dockerfile
       args:
-        VERSION: 1.18
-
+        VERSION: 1.19
```

Comment:
1.18? 1.19 is defined bellow

Reply:
@pchico83 This is for the `okteto/golang:1` tag which should always point to the latest `1.x`

Reply:
ah, perfect